### PR TITLE
Update networking-recipe for ES2K target

### PR DESCRIPTION
**Pull request by ffoulkes**
Added support for the Intel Infrastructure Processing Unit (IPU) E2000 ASIC.

### CODE REVIEW

File: .github/workflows/build_and_test.yml
Comment (5abeel) [inline .github/workflows/build_and_test.yml:22]: Probably a question for @mgaravin -- how would this work for MEV/ES2K target? Would CI builds still be supported through the Github runners?

**Review by nupuruttarwar — APPROVED** [review]
LGTM

File: .github/workflows/build_and_test.yml
Comment (ffoulkes) [reply .github/workflows/build_and_test.yml:22]: No clue. We'll have to wait for the CI/CD boffins to weigh in.

Comment (mgaravin) [reply .github/workflows/build_and_test.yml:22]: Yes, we have Intel shared GitHub runners provided by 1source team! I'm trying to make that work, if not i have to add dedicated "self-hosted" runner and execute the github actions.

Pls. note: Self-host runner can't run all opensource github actions ( as few actions as expected to run only in opensource environment)! have to check and do modifications in innersource based on our need!

Comment (mgaravin) [reply .github/workflows/build_and_test.yml:22]: And, probably, we should not upstream innersource github actions to opensource, as it was customized to run only on innersource environment

Comment (ffoulkes) [reply .github/workflows/build_and_test.yml]: I have backed out the changes to the github actions

**Comment by ffoulkes** [comment]
We are currently blocked, waiting for approval to open-source the MEV changes.

**Review by ffoulkes — COMMENTED** [review]
Reviewed in order to create a summary of changes made for ES2K.

Added a number of comments along the way.

**Pull request closed without merging**
